src/schema/v2/previewSavedSearch.ts

```diff
@@ -99,13 +101,70 @@ const PreviewSavedSearchType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(
         new GraphQLList(new GraphQLNonNull(SearchCriteriaLabel))
       ),
-      resolve: (_) => {
-        return mockSuggestedFilters
+      resolve: async (_root, _args, { filterArtworksLoader }) => {
+        const suggestedFiltersByArtist: SearchCriteriaLabel[][] = await Promise.all(
+          _root.artistIDs.map((artistSlug) =>
+            getSuggestedFiltersByArtistSlug(artistSlug, filterArtworksLoader)
+          )
+        )
+
+        return _.chain(suggestedFiltersByArtist)
+          .flatten()
+          .uniqBy((searchCriteria) => JSON.stringify(searchCriteria))
+          .value()
       },
     },
   }),
 })
 
+const getMostPopularField = (aggregation: {
```

Comment:
**suggestion:** The "field" naming here might be unintentionally confusing. I'd opt for "most popular value" or "most popular option" instead.